.github/workflows/ci-workflow.yml

```diff
@@ -72,3 +72,37 @@ jobs:
       run: npm run unit
     - name: Run Integration
       run: npm run integration
+
+  windows-latest:
+    runs-on: windows-latest
+
+    # Node 16+ should eventually bundle node-gyp>=8.4.0 to be compatible with Server 2022.
+    # Once compatible, can remove node-gyp upgrade.
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Get Node Version
+      id: node_version
+      run: echo "::set-output name=version::$(node --version | sed 's/^.//')"
+
+    # https://github.com/nodejs/node-gyp/blob/master/docs/Updating-npm-bundled-node-gyp.md#windows
+    - name: Upgrade node-gyp in npm
+      run: |
+        cd "C:\hostedtoolcache\windows\node\${{ steps.node_version.outputs.version }}\x64"
```

Comment:
There's probably a better way to do this.

On quick scan, I didn't find an obvious place to pull the path from.

Trying to grab similar to the version... `where path` returned nothing. `which path` returned a Linux style path `/c/hostedtoolcache/ ...` so here's what I landed with to prove it out.